apps/rush-lib/src/logic/taskRunner/ProjectTask.ts

```diff
@@ -58,24 +63,26 @@ export class ProjectTask implements ITaskDefinition {
   private _hasWarningOrError: boolean;
   private _rushProject: RushConfigurationProject;
   private _rushConfiguration: RushConfiguration;
-  private _commandToRun: string;
+  private _commandsToRun: IScriptCommand[];
   private _packageChangeAnalyzer: PackageChangeAnalyzer;
   private _packageDepsFilename: string;
 
   public constructor(options: IProjectTaskOptions) {
     this._rushProject = options.rushProject;
     this._rushConfiguration = options.rushConfiguration;
-    this._commandToRun = options.commandToRun;
     this.isIncrementalBuildAllowed = options.isIncrementalBuildAllowed;
     this._packageChangeAnalyzer = options.packageChangeAnalyzer;
     this._packageDepsFilename = options.packageDepsFilename;
-}
+
+    this._commandsToRun = options.commandsToRun;
+    if (this._commandsToRun.length < 1) {
+      throw Error("No commands were defined for the task.");
```

Comment:
_commandsToRun could also be empty if --skip parameter has all parameters listed as values, right?